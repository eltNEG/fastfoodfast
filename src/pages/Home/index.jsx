import React from 'react';
import { Grid } from 'semantic-ui-react';
import HomeHero from '../../components/HomeHero';
import HomeContent from '../../components/HomeContent';

export default () => (
  <Grid className="main home">
    <Grid.Row>
      <HomeHero />
    </Grid.Row>
    <Grid.Row>
      <HomeContent />
    </Grid.Row>
  </Grid>
);
