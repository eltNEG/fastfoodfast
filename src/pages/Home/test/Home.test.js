import React from 'react';
import { shallow } from 'enzyme';
import Home from '..';

describe('Home snapshot', () => {
  it('should match Home.jsx snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
