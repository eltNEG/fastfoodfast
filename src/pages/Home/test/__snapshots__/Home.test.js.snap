// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot should match Home.jsx snapshot 1`] = `
<div
  className="main home"
>
  Home
</div>
`;
